src/pages/error404/error404.js

```diff
@@ -0,0 +1,21 @@
+import Handlebars from 'handlebars';
+import template from './error404.tmpl';
+import './error404.scss';
+import '../../partials/floatingContainer'
```

Comment:
Если используем в проекте `;`, то лучше стараться использовать везде. Для поддержания чистоты кода
```suggestion
import '../../partials/floatingContainer';
```